tqecjs/src/workspace/plaquettes/PlaquetteClass.js

```diff
@@ -368,8 +369,9 @@ export default class Plaquette extends Graphics {
 		this.controlPanel.addChild(this.colorButton);
 	};
 
-	clearPlaquette = () => {
+	initializeClearButton = () => {
 		this.clearButton.on('click', (_event) => {
+			console.log("Destroying plaquette")
```

Comment:
The destroying feature still needs a bit of work and I've spun it out into an issue: [#29](https://github.com/QCHackers/tqec/issues/24)